modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextStore.java

```diff
@@ -77,9 +77,17 @@ void storeDurableSubscription(AndesSubscription subscription)
      *
      * @param subscription The subscriptoin object to update
      * @throws AndesException
+     * @return If a subscription was updated return 1, else return 0
      */
-    void updateDurableSubscription(AndesSubscription subscription)
-		    throws AndesException;
+    int updateDurableSubscription(AndesSubscription subscription) throws AndesException;
```

Comment:
query.executeUpdate() directy returns an int. We don't need to do another if and convert this to a boolean.